source/includes/v2/_odsql.md

```diff
@@ -470,17 +470,36 @@ GEOMETRY(field_name, GEOM'<geometry>', DISJOINT)
 GEOMETRY(field_name, GEOM'<geometry>', WITHIN)
 ```
 
-The geometry function limits the result set to a geographical area defined by a polygon.
+The geometry function limits the result set to a geographical area defined by a geometry.
 
-This polygon must be defined with both:
+This function only supports geo_shape field for the moment. [Polygon function](#polygon-function) can be used to filter on a geo_point field.
```

Comment:
I would say something like "This function is used on a geo_shape field. It should not be confused with the polygon function, which is used on a geo_point field." What do you think? This removes any doubt on what each of them does.